aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -315,5 +315,32 @@ Tensor& matmul_out(Tensor &result, const Tensor & tensor1, const Tensor & tensor
   return result;
 }
 
+Tensor frobenius_norm(const Tensor& self) {
+  IntList dim = {0, 1};
+  return at::native::frobenius_norm(self, dim);
 }
+
+Tensor frobenius_norm(const Tensor& self, IntList dim, bool keepdim) {
+  AT_CHECK(
+      dim.size() <= 2,
+      "Improper number of chosend dimensions to do frobenius norm.");
+  if (dim.size() == 0) {
+    Tensor result = self.type().tensor();
+    return at::native::norm_out(result, self, 2, self.dim(), keepdim);
+  }
+  if (dim.size() == 1) {
+    return at::native::norm(self, 2, dim[0], keepdim);
```

Comment:
All these native calls break autograd. Please use the top namespace ones, e.g., `at::norm`